fea-rs/src/compile/compile_ctx.rs

```diff
@@ -950,13 +950,20 @@ impl<'a> CompilationCtx<'a> {
     fn add_contextual_pos_rule(&mut self, node: &typed::Gpos8) {
         let backtrack = self.resolve_backtrack_sequence(node.backtrack().items());
         let lookahead = self.resolve_lookahead_sequence(node.lookahead().items());
+        let trailing_value_record = node.trailing_value_record();
+        if trailing_value_record.is_some() {
+            // sanity check for weird gpos special case
+            debug_assert_eq!(node.input().items().count(), 1,);
```

Comment:
add a message?

Reply:
I removed it so that this could be formatted on a single line. If I ever hit this the panic will have the line number, so it'll be easy to track down.